packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -253,14 +253,84 @@ void main() {
       ),
     );
 
-    await tester.fling(find.text('X'), const Offset(0.0, 100.0), 1000.0);
+    await tester.fling(find.text('X'), const Offset(0.0, 200.0), 1000.0);
```

Comment:
Can you explain why this needed to change?

Reply:
I believe it is because of the bounceback issue that I described in #138848.

Reply:
It looks like it came from #69783. It seems intentional to match the native behavior, but it may have changed since then.